src/components/Sum.jsx

```diff
@@ -1 +1,20 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```
in this case better to write props in one line. also you can write default props here